### DISCUSSION TITLE
"Do you wish to see all Possibilities?" triggered with group description colors

### DISCUSSION THREAD

**Discussion by har-si** [marlonrichert/zsh-autocomplete#646] [Q&A]
I added the below line to my `.zshrc` to change the color of the group description in my completion (described in #618 ):

`zstyle ':completion:*:*:descriptions' format '%F{blue}%d%f'`

The group description color changes to Blue, but it triggered the "Do you wish to see all __ Possibilities?" prompt which is very annoying when typing your commands.

![autocompleteBug](https://github.com/marlonrichert/zsh-autocomplete/assets/122127381/a836fec3-11be-44eb-9a40-e633200565bb)

When I removed this `zstyle ':completion:*:*:descriptions' format '%F{blue}%d%f'` line in my `.zshrc`, everything went back to its normal behavior (no more annoying prompt).

I have read from previous discussion #625 that this is a bug. Is it possible to disable the "Do you wish to see all __ Possibilities?" prompt while having colored group descriptions?

btw, I installed zsh-autocomplete manually and my zsh version is 5.8 (x86_64-debian-linux-gnu).

Thanks a lot!

**Comment by marlonrichert** [comment]
That is definitely a bug. I created an issue for it: #654.

**Comment by jul-sh** [comment]
try this config in zshrc `zstyle ':completion:*' menu yes select`
